pom.xml

```diff
@@ -0,0 +1,506 @@
+<!--        <module>aad/spring-cloud-azure-starter-active-directory-b2c/aad-b2c-resource-server</module>-->
+<!--        <module>aad/spring-security</module>-->
+      </modules>
+      <properties>
+        <maven.compiler.source>17</maven.compiler.source>
+        <maven.compiler.target>17</maven.compiler.target>
+        <java.version>17</java.version>
+        <spring.boot.version>3.0.0-RC2</spring.boot.version>
+        <version.spring.cloud>2022.0.0-RC2</version.spring.cloud>
+        <version.spring.cloud.azure>6.0.0-beta.4</version.spring.cloud.azure>
+        <version.thymeleaf-extras-springsecurity6>3.1.1.RELEASE</version.thymeleaf-extras-springsecurity6>
+        <mockito.version>4.8.1</mockito.version>
+      </properties>
+      <dependencyManagement>
+        <dependencies>
+          <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-dependencies</artifactId>
+            <version>${spring.boot.version}</version>
+            <type>pom</type>
+            <scope>import</scope>
+          </dependency>
+          <dependency>
+            <groupId>org.springframework.cloud</groupId>
+            <artifactId>spring-cloud-dependencies</artifactId>
+            <version>${version.spring.cloud}</version>
+            <type>pom</type>
+            <scope>import</scope>
+          </dependency>
+          <dependency>
+            <groupId>com.azure.spring</groupId>
+            <artifactId>spring-cloud-azure-dependencies</artifactId>
+            <version>${version.spring.cloud.azure}</version>
+            <type>pom</type>
+            <scope>import</scope>
+          </dependency>
+          <dependency>
+            <groupId>org.thymeleaf.extras</groupId>
+            <artifactId>thymeleaf-extras-springsecurity6</artifactId>
+            <version>${version.thymeleaf-extras-springsecurity6}</version>
+          </dependency>
+        </dependencies>
+      </dependencyManagement>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-maven-plugin</artifactId>
+            <version>${spring.boot.version}</version>
+          </plugin>
+        </plugins>
+      </build>
+      <repositories>
+        <repository>
+          <id>spring-milestones</id>
+          <name>Spring Milestones</name>
+          <url>https://repo.spring.io/milestone</url>
+          <snapshots>
+            <enabled>false</enabled>
+          </snapshots>
+        </repository>
+      </repositories>
+      <pluginRepositories>
+        <pluginRepository>
+          <id>spring-milestones</id>
+          <name>Spring Milestones</name>
+          <url>https://repo.spring.io/milestone</url>
+          <snapshots>
+            <enabled>false</enabled>
+          </snapshots>
+        </pluginRepository>
+      </pluginRepositories>
+    </profile>
+    <profile>
+      <id>buildpack-4x</id>
```

Comment:
For profiles except `spring-cloud-azure-4` and `spring-cloud-azure-6`, the `modules` are empty, is it right?

Reply:
Is it valid to active `buildpack-4x` and `spring-cloud-azure-6` at the same time? It should be written down in README.md, otherwise our customers will be confused.

Reply:
No, I have updated the doc for Spring native parts in *spring-native/storage-blob-native/README.md*; the extra 6.x profiles are used to simulate the profile `native` and `nativeTest` in `spring-boot-starter-parent`, currently, we will not recommend users to use these features.